docs/user_guide/install_client.md

```diff
@@ -1,70 +1,128 @@
+1. Obtain the source code of Dragonfly.
 
-    - [MacOS 64-bit](https://github.com/dragonflyoss/Dragonfly/releases/download/v0.4.3/df-client_0.4.3_darwin_amd64.tar.gz): `https://github.com/dragonflyoss/Dragonfly/releases/download/v0.4.3/df-client_0.4.3_darwin_amd64.tar.gz`
+    ```sh
+    git clone https://github.com/dragonflyoss/Dragonfly.git
+    ```
 
-2. Unzip the package.
+2. Enter the project directory.
 
-    ```bash
-    # Replace `xxx` with the installation directory.
-    tar -zxf df-client_0.4.3_linux_amd64.tar.gz -C xxx
+    ```sh
+    cd Dragonfly
     ```
 
-3. Add the directory of `df-client` to your `PATH` environment variable to make sure you can directly use `dfget` and `dfdaemon` command.
+3. Build the Docker image.
 
-    ```bash
-    # Replace `xxx` with the installation directory.
-    # Execute or add this line to ~/.bashrc
-    export PATH=$PATH:xxx/df-client/
+    ```sh
+    TAG="1.0.0"
+    make docker-build-client DF_VERSION=$TAG
+    ```
+
+4. Obtain the latest Docker image ID of the `dfclient`.
+
+    ```sh
+    docker image ls | grep 'dfclient' | awk '{print $3}' | head -n1
+    ```
+
+### Start the dfdaemon
+
+**NOTE:** You should prepare the [config files](../config) which should locate under `/etc/dragonfly` by default.
+
+```sh
+version=1.0.0
+# Replace ${supernode} with your own supernode node with format `ip:port=weight`.
+SUPERNODE=$supernode
+docker run -d --name dfclient --restart=always -p 65001:65001 -v $HOME/.small-dragonfly:/root/.small-dragonfly -v /etc/dragonfly:/etc/dragonfly dragonflyoss/dfclient:$version --node $SUPERNODE
+```
+
+## Procedure - When Deploying with Physical Machines
+
+### Get dfclient executable file
+
+1. Download a binary package of the SuperNode. You can download one of the latest builds for Dragonfly on the [github releases page](https://github.com/dragonflyoss/Dragonfly/releases).
+
+    ```sh
+    version=1.0.0
+    wget https://github.com/dragonflyoss/Dragonfly/releases/download/v$version/Dragonfly_$version_linux_amd64.tar.gz
     ```
 
-## Installing from the Source Code
+2. Unzip the package.
 
-You can also install from the source code.
+    ```bash
+    # Replace `xxx` with the installation directory.
+    tar -zxf Dragonfly_1.0.0_linux_amd64.tar.gz -C xxx
+    ```
 
-**Note:** You must have started Docker.
+3. Move the `dfget` and `dfdaemon` to your `PATH` environment variable to make sure you can directly use `dfget` and `dfdaemon` command.
 
-### Installing in $HOME/.dragonfly
+Or you can build your own dfclient executable files.
 
 1. Obtain the source code of Dragonfly.
 
     ```sh
     git clone https://github.com/dragonflyoss/Dragonfly.git
     ```
 
-2. Enter the target directory.
+2. Enter the project directory.
 
     ```sh
     cd Dragonfly
     ```
 
-3. Build `dfdaemon` and `dfget`.
+3. Compile the source code.
```

Comment:
What's wrong with `build dfdaemon and dfget`?  I think the previous one is better. What about build the executables/binaries?